docs/core/introduction.md

```diff
@@ -6,100 +6,59 @@ ms.custom: "updateeachrelease"
 ---
```

Comment:
Delete `(and .NET Core)` in the title above?

Reply:
I assume it is there for SEO type reasons.

@gewarren?

Reply:
I think we can delete it. Not sure why it's there.

Reply:
Done